tools/static-code-analysis/pmd/suppressions.properties

```diff
@@ -3,3 +3,8 @@ org.eclipse.smarthome.io.console.karaf.internal.OSGiConsole=SystemPrintln
 org.eclipse.smarthome.io.console.rfc147.internal.CommandWrapper=SystemPrintln
 org.eclipse.smarthome.io.console.rfc147.internal.OSGiConsole=SystemPrintln
 org.eclipse.smarthome.core.internal.events.ThreadedEventHandler=EmptyIfStmt
+org.eclipse.smarthome.config.xml.ConfigDescriptionConverter=AvoidCatchingNPE
+org.eclipse.smarthome.io.rest.core.internal.thing.ThingResource=AvoidCatchingNPE
```

Comment:
Shouldn't this be easy to fix? At which places do we expect that an NPE is thrown?